### ISSUE TITLE
Release 1.1 planning

### ISSUE THREAD

**Issue by koogoro** [python/mypy#14730]
I'm planning to make release 1.1 by the end of February.

Please post here any issues you'd like to see fixed in the release, or PRs you'd like to have merged into the release branch

**Comment by sterliakov** [comment]
https://github.com/python/mypy/pull/12572

This technique is used in django-stubs to mark a call as invalid without resorting to plugin. `NoReturn` is currently handled almost everywhere, but ignored for `__setitem__` for some reason.

**Comment by Avasam** [comment]
It's small, but I'd really like https://github.com/python/mypy/pull/14437 for QoL improvement in stubtest.
Another typeshed QoL improvement (and mypy bug with aliased properties) that would be nice is https://github.com/python/mypy/issues/13975
and stubtest name mangling weirdness: https://github.com/python/mypy/issues/14560

Out of all my issues, these are the 3 I care more about right now and would also help typeshed.

**Comment by cdce8p** [comment]
Fix for a crash with `dataclass_transform`
* #14734

**Comment by hauntsaninja** [comment]
+1 to getting in the currently open dataclass_transform PRs

**Comment by ichard26** [comment]
FYI the [`release-1.1` branch does exist now](release-1.1) contrary to the OP. (_I would edit the issue description, but I'm merely a triager :slightly_smiling_face:_)

**Comment by koogoro** [comment]
It looks like #14437 was included in the base of the release branch, and #14695 and #14752 have been cherry-picked.

**Comment by hauntsaninja** [comment]
Could cherry pick https://github.com/python/mypy/pull/14792

**Comment by tmke8** [comment]
Support for `@override` would be nice. The following PR implements it:
- #14609
But there was an open question about the interaction with `@overload`.

**Comment by koogoro** [comment]
Cherry-picked #14792. It might be better to wait for the next release for a new feature, since the release is already a bit late at this point.

**Comment by AlexWaygood** [comment]
Argh, you might have to cherry-pick these two to fix the mypyc build on the 1.1 branch — sorry:
- #14781
- #14787

**Comment by AlexWaygood** [comment]
Here's a PR to fix the mypyc build on the 1.1. branch:

- #14820

**Comment by Viicos** [comment]
By any chance do you think this one could be included as well?

- https://github.com/python/mypy/pull/14471

It has been requested by a couple of people, and unfortunately I'm kind of stuck on this one. If you think it is fixable without too much hassle I'd be grateful to get help on this one!

(I'm posting it here as it might be considered as a feature and not a fix according to the new release versioning scheme).

**Comment by koogoro** [comment]
@Viicos Let's wait on this change until the next release. These are intended to happen monthly, so it shouldn't be too long until the next one. In general, I'd prefer not to cherry-pick changes onto the release at this point unless it's fixing a bug that would be introduced by this release.

**Comment by Viicos** [comment]
> @Viicos Let's wait on this change until the next release. These are intended to happen monthly, so it shouldn't be too long until the next one. In general, I'd prefer not to cherry-pick changes onto the release at this point unless it's fixing a bug that would be introduced by this release.

Sure thanks for the answer. There's no hurry for this one anyway

**Comment by koogoro** [comment]
Release made! Here's the blog post: https://mypy-lang.blogspot.com/2023/03/mypy-111-released.html
(The version number is 1.1.1 since 1.1.0 was not building on wheels.)
I'll keep this open for a little bit to track potential bugs introduced in this release that we'd want to create an expedited 1.1.2 release for.

**Comment by AlexWaygood** [comment]
We've just had a regression reported over at typeshed:

- https://github.com/python/typeshed/issues/9854

This regression was caused by https://github.com/python/typeshed/pull/9362. It's already been fixed on the typeshed `main` branch via https://github.com/python/typeshed/pull/9758, but the fix didn't make it into 1.1.1. (This is my error.)

**Comment by koogoro** [comment]
https://github.com/python/mypy/issues/14853 might be a candidate for a point release if the fix is simple enough.

**Comment by jchalupka-pan** [comment]
> #14853 might be a candidate for a point release if the fix is simple enough.

@koogoro The issue has been fixed. Not hurrying up of course, I just don't know if you have been notified somehow :)

**Comment by wesleywright** [comment]
I've got a couple of dataclass_transform fixes inflight:

- https://github.com/python/mypy/pull/14870 (not sure how big this will be when it comes out of Draft, but hopefully small?)
- https://github.com/python/mypy/pull/14878 is a simple and straightforward fix

**Comment by AlexWaygood** [comment]
This fixes a (minor) regression; could be worth backporting:

- #14881
